releases/v0.2.0.md

```diff
@@ -0,0 +1,71 @@
+# Release Notes for v0.2.0
+Confidential Containers has adopted a six-week release cadence.
+This is our first release on this schedule.
+This release mainly features incremental improvements to our build system and tests
+as well as minor features, adjustments, and cleanup.
+
+Please see the [quickstart guide](../quickstart.md) for details on how to try out Confidential Containers
+
+## What's new
+
+- Kata CI uses existing Kata tooling to build components
+- Kata CI caches build environments for components
+- Incremental advances in signature support
+- KBS no longer required to use unencrypted images with SEV.
+
+## Hardware Support
+Confidential Containers is tested with attestation on the following platforms:
+- Intel TDX
+- AMD SEV
+
+The following platforms are untested or partially supported:
+- AMD SEV-ES
+- IBM Z SE
+
+The following platforms are in development:
+- Intel SGX
+- AMD SEV-SNP
+
+## Limitations
+
+The following are known limitations of this release:
+
+- Platform support is currently limited, and rapidly changing
+  * S390x is not supported by the CoCo operator
+  * AMD SEV-ES has not been tested.
+  * AMD SEV does not support container image signature validation.
+- Attestation and key brokering support is still under development
+  * The disk-based key broker client (KBC) is used when there is no HW support, but is not suitable for production (except with encrypted VM images).
+  * Currently, there are two KBS that can be used:
+    - simple-kbs:  simple key broker service (KBS) for SEV(-ES).
+    - [Verdictd](https://github.com/inclavare-containers/verdictd): An external project with which Attestation Agent can conduct remote attestation communication and key acquisition via EAA KBC
+  * The full-featured generic KBS and the corresponding KBC are still in the development stage.
+  * For developers, other KBCs can be experimented with.
+  * AMD SEV must use a KBS even for unencrypted images.
+- The format of encrypted container images is still subject to change
+  * The oci-crypt container image format itself may still change
+  * The tools to generate images are not in their final form
+  * The image format itself is subject to change in upcoming releases
+  * Image repository support for encrypted images is unequal
```

Comment:
And now due to https://github.com/kata-containers/kata-containers/issues/5582, now `S390x is not support by cosign-signed image verification`

Reply:
@Xynnn007 - hey Ding, the situation is worse than this I'm afraid - due to the incompatibility with the `ring` crate and `s390x` we aren't able to build the kata-agent for `s390x` at all, so if we want to add co-sign support this release (which I'm not against), and document the restriction in the release notes, we'd just need to say that `s390x` isn't working at all for this release.

Reply:
I got that. So due to this issue (incompatibility with the `ring` crate and `s390x`), what should we do now? I'm not sure, are there two different ways?
- Say that this release do not support `s390x` at all (for we can not build kata-agent now)
- Put a fix in `image-rs` to add a switch `cosign` verification (feature in `Cargo.toml`), and add compile configuration in `kata-agent` to disable cosign when the arch is `s390x`. Also, some CI fixes when the arch is `s390x`.

Reply:
Or we can not add `cosign` this release? CC @peterzcst @arronwy

Reply:
I think that due to `s390x` not being supported, the community will vote for
> Say that this release do not support s390x at all (for we can not build kata-agent now)

It is a bit unfortunately timing which that due to delayed backporting of an `s390x` from `main` into `CCv0` we missed spotting the failure in the PR that bumped image-rs, but as long as the community continue to try and help fix this for 0.3 (which I'll recognise you have been doing so far 🙇), then I won't suggest holding back this feature from the release.

A lesson for the future is that we need to ensure that we get some s390x tests as part of the image-rs CI like we've done for the attestation-agent. That's been on my list for a while though and is IBM's responsibility to drive.

Reply:
Talking to @hbrueckner - we've been discussing the idea of pulling in an arch specific version of `image-rs` into the kata-agent. Then we could pull the `0.2` tag for non-s390x arch and the commit before the `ring` dependency got pulled in for s390x therefore losing co-sign support for s390x, but enabling compilation.

I don't like splitting the code like this and it would obviously not be supported, but is an idea to avoid a s390x regression for 0.2?

Reply:
I'm fine with this approach.

Reply:
I think that now we have https://github.com/kata-containers/kata-containers/pull/5600 merged, we might be back to Ding's original suggest of documenting the s390x limitation, so something like:
> Upstream Kata Agent cannot build for `s390x`, meaning that `s390x` misses some newer features, such as cosign support.

Reply:
Oh well, (shamelessly stolen from the wrong place): with https://github.com/kata-containers/kata-containers/pull/5600 merged, kata-agents builds again. So it should more read like:

> s390x does not support container image signature with cosign in this release.

Reply:
Thanks for the clarification!